doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -205,9 +423,333 @@ Mail chapel-developers@ for review and merge
   have intentionally made the process as lightweight as possible, so it can be
   used for changes big and small alike.
```

Comment:
This last bullet no longer belongs in this section. Not sure it's really necessary anywhere.